tutorials/julia-jupyter-notebook-server/index.md

```diff
@@ -0,0 +1,138 @@
+
+Install the Jupyter Python modules if necessary.
+```sh
+pip3 install jupyter
+```
+Generate a hashed version of your desired password.
+```sh
+PASSWD=[YOUR DESIRED PASSWORD]
+HASHED_PASSWD=$(python3 -c "from notebook.auth import passwd; print(passwd(\"${PASSWD}\"))")
+```
+If you can't install the Jupyter Python modules you can use the hashed version of the password: ```$$nTh3b@nc```. **This is not the recommended approach. If you use it change the password on your notebook server when you login the first time.**
+```sh
+HASHED_PASSWD='sha1:8f334ff5f862:c19298d6e4f03fe9ec6e6a5c127927c86d47ec2a'
+```
+Update ```jupyter-config.sh``` with the hashed version of your password.
+```sh
+sed -i 's/HASHED_PASSWD/'"${HASHED_PASSWD}"'/' jupyter-config.sh
+```
+
+## Verify your configuration
+Generate a Terraform plan
+```sh
+terraform plan -out tf.plan -auto-approve
+```
+The terminal output describes the resources that Terraform will create/configure.
+
+If ```use_acme_cert``` is true (the default) the output will include
+* acme_certificate.certificate: check that the *common_name* field contains the correct FQDN for your notebook server
+* acme_registration.reg: check that the *email_address* field contains the correct email address
+* google_compute_firewall.jupyter-server: check that the *target_tags.nnnnnnnnnn* field is set to *jupyter-server-[your server name]*
+* google_compute_instance.nbs_acme_cert: check that the *tags.nnnnnnnnnn* field is set to *jupyter-server-[your server name]*
+* google_dns_record_set.nbs_acme_cert: check that the *name* field contains the correct FQDN for your notebook server
+* tls_private_key.private_key
+
+If you set ```use_acme_cert``` to false only these resources will be created/configured
+* google_compute_firewall.jupyter-server: check that the *target_tags.nnnnnnnnnn* field is set to *jupyter-server-[your server name]*
+* google_compute_instance.nbs_self_signed_cert: check that the *tags.nnnnnnnnnn* field is set to *jupyter-server-[your server name]*
+* google_dns_record_set.nbs_self_signed_cert: check that the *name* field contains the correct FQDN for your notebook server
+
+## Create the notebook server
+To create the notebook server type
+```sh
+terraform apply tf.plan
+```
+The terminal output logs Terraform's progress as it executes the plan you generated earlier. When it completes you will see:
+```sh
+Apply complete! Resources: 6 added, 0 changed, 0 destroyed.
+```
+Installing the Juila kernel takes approximately 10 minutes. Therefore even though the Compute Engine instance is running you won't be able to immediately connect to the notebook server. Wait 10 minutes and then proceed.
+
+## Log into the notebook server
+The URL for your notebook server has the form: ```https://[your server name].[your domain]:8089```, *e.g.*, ```https://dobbs.sierramadre.net:8089```. When you navigate there with your browser you should see the Jupyter login screen.
```

Comment:
"When you navigate there with your browser you should see the Jupyter login screen." -> "When you navigate there with your browser, you should see the Jupyter login screen."

Reply:
fixed in Update index.md (15)